ray_provider/xcom/ray_backend.py

```diff
@@ -120,6 +120,9 @@ def get_actor(self, identifier, allow_new):
             else:
                 raise
 
+    def get_store(self, identifier):
+        return self.get_actor(identifier, allow_new=False).store
```

Comment:
hmm, this won't work since we don't allow you to access class variables of an actor